src/pages/Encounters/EncounterList.tsx

```diff
@@ -440,7 +434,7 @@ export function EncounterList({
               </div>
 
               {/* Status Filter - Desktop */}
-              <div className="hidden md:flex items-center">
+              <div className="hidden md:flex items-center py-1 mt-3">
```

Comment:
Same with these. You can derive the data-cy values from status (adjust the test cases if need be).